### PR TITLE
Ks 2022 08 merge content types

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,9 @@
 /static/
 /website_wagtail/static
 media/
-db.sqlite3
+db.sqlite3*
+*.sqlite3
+.idea/
```
Comment (khamui) [inline .gitignore:9]: out of curiosity: what is this `.idea` folder for?

**Review by khamui — APPROVED** [review]
lgtm. But leaving someone with expertise to merge.

File: .gitignore (hunk 1 above)
Comment (Rineee) [reply .gitignore:9]: https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-
I think its pycharm specific, but say that we have it in other gitignore files, so also added it here..

**Comment by fuzzylogic2000** [comment]
I fixed the conflict and also the migration, which needed to be changed from 7 to 8. Will merge once this is run.

**Pull request merged**